src/pages/404.tsx

```diff
@@ -0,0 +1,46 @@
+import { getMessages } from '../utils/locale';
+import { GetStaticProps } from 'next';
+import Head from 'next/head';
+import { FormattedMessage as Msg } from 'react-intl';
+import { Link as AdobeLink, Content, Flex, Heading } from '@adobe/react-spectrum';
```

Comment:
Is it necessary to import `Link as AdobeLink` here, or could it's default name be used?

In other places, where `Link` from Spectrum has collided with `Link` from NEXT, we have renamed the NEXT one instead. But here there doesn't seem to be a collision at all?